packages/angular/.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
+  "root": true,
+  "ignorePatterns": [
+    "**/stories/**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@angular-eslint/recommended",
+        "plugin:@angular-eslint/template/process-inline-templates"
+      ],
+      "rules": {
+        "@angular-eslint/directive-selector": [
+          "error",
+          {
+            "type": "attribute",
+            "prefix": "app",
```

Comment:
This should probably change? `sl`? Or should it be different to prevent conflicts? `sla`?

Reply:
Maybe `sla` will be fine in this case